pywr-schema/src/nodes/core.rs

```diff
@@ -1180,6 +1514,38 @@ mod tests {
         run_all_solvers(&model, &["clp"], &expected_outputs);
     }
 
+    #[cfg(feature = "core")]
+    fn link_with_soft_min_str() -> &'static str {
+        include_str!("../test_models/link_with_soft_min.json")
+    }
+
+    #[test]
+    #[cfg(feature = "core")]
+    fn test_link_with_soft_min() {
```

Comment:
Tests look good to me. I guess we're just missing some expected output. Happy to do that in a follow-up though.

Reply:
Yes that's still missing. I'll open a separate issue